src/Pimple/Tests/PimpleTest.php

```diff
@@ -115,6 +115,18 @@ public function testOffsetGetValidatesKeyIsPresent()
         echo $pimple['foo'];
     }
 
+    /**
+     * @test
```

Comment:
Instead of using the `@test` tag, you can name your test normally `testOfssetGet...` as the `@group legacy` annotation is enough to mark it as legacy.